resolver/sudn_resolver.go

```diff
@@ -0,0 +1,145 @@
+package resolver
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/0xERR0R/blocky/model"
+	"github.com/miekg/dns"
+)
+
+const (
+	sudnTest      = "test."
+	sudnInvalid   = "invalid."
+	sudnLocalhost = "localhost."
+)
+
+func sudnArpaSlice() []string {
+	return []string{
+		"10.in-addr.arpa.",
+		"21.172.in-addr.arpa.",
+		"26.172.in-addr.arpa.",
+		"16.172.in-addr.arpa.",
+		"22.172.in-addr.arpa.",
+		"27.172.in-addr.arpa.",
+		"17.172.in-addr.arpa.",
+		"30.172.in-addr.arpa.",
+		"28.172.in-addr.arpa.",
+		"18.172.in-addr.arpa.",
+		"23.172.in-addr.arpa.",
+		"29.172.in-addr.arpa.",
+		"19.172.in-addr.arpa.",
+		"24.172.in-addr.arpa.",
+		"31.172.in-addr.arpa.",
+		"20.172.in-addr.arpa.",
+		"25.172.in-addr.arpa.",
+		"168.192.in-addr.arpa.",
+	}
+}
+
+type SudnResolver struct {
```

Comment:
Was a bit confused about naming. 😕  
Found later as comment: Sudn = Special-Use Domain Names. Maybe we should rename the struct to "SpecialUseDomainNamesResolver" to make it more concise?

Reply:
Yeah thought this would be a very long name especially as file name.
Only renaming the struct would be more readable without obscenely long filenames.

What do you think?

Reply:
> Yeah thought this would be a very long name especially as file name. Only renaming the struct would be more readable without obscenely long filenames.
> 
> What do you think?

Sure, renaming the struct only would be nice. 👍